libs/fixture-generators/src/generate-election-package/generate_election_package.ts

```diff
@@ -0,0 +1,229 @@
+} from '@votingworks/design-backend';
+import {
+  createPlaywrightRenderer,
+  renderAllBallotsAndCreateElectionDefinition,
+  Renderer,
+  vxDefaultBallotTemplate,
+} from '@votingworks/hmpb';
+import {
+  BallotType,
+  DEFAULT_SYSTEM_SETTINGS,
+  Election,
+  ElectionDefinition,
+  ElectionPackage,
+  ElectionPackageFileName,
+  ElectionPackageMetadata,
+  formatElectionHashes,
+  mergeUiStrings,
+  UiStringsPackage,
+} from '@votingworks/types';
+import {
+  BooleanEnvironmentVariableName,
+  isFeatureFlagEnabled,
+} from '@votingworks/utils';
+import { sha256 } from 'js-sha256';
+import JsZip from 'jszip';
+import { writeFile } from 'node:fs/promises';
+import path from 'node:path';
+import { Readable } from 'node:stream';
+
+// In order for the zip files generated to hash to the same value when the contents
+// are the same we need to make sure the date on the files is always kept static.
+const FIXTURES_FILE_DATE = new Date('2024-12-01T00:00:00Z');
+
+function getBallotLanguageConfigs(): BallotLanguageConfigs {
+  const translationsEnabled = isFeatureFlagEnabled(
+    BooleanEnvironmentVariableName.ENABLE_CLOUD_TRANSLATION_AND_SPEECH_SYNTHESIS
+  );
+  return translationsEnabled
+    ? Object.values(LanguageCode).map(
+        (l): BallotLanguageConfig => ({ languages: [l] })
+      )
+    : [{ languages: [LanguageCode.ENGLISH] }];
+}
+/**
+ * Generates all of the new strings for an election.
+ */
+export async function getAllStringsForElection(
+  election: Election,
+  translator: GoogleCloudTranslatorWithoutCache,
+  ballotLanguageConfigs: BallotLanguageConfigs
+): Promise<[UiStringsPackage, UiStringsPackage, UiStringsPackage]> {
+  const appStrings = await translateAppStrings(
+    translator,
+    'latest',
+    ballotLanguageConfigs
+  );
+  const hmpbStrings = await translateHmpbStrings(
+    translator,
+    ballotLanguageConfigs
+  );
+  const electionStrings = await extractAndTranslateElectionStrings(
+    translator,
+    election,
+    ballotLanguageConfigs
+  );
+
+  return [appStrings, hmpbStrings, electionStrings];
+}
+
+/**
+ * Wrapper helper to generate an election definition with grid layouts from a given election.
+ */
+export async function generateElectionDefinitionForHmpbs(
+  renderer: Renderer,
+  election: Election
+): Promise<ElectionDefinition> {
+  const { electionDefinition } =
+    await renderAllBallotsAndCreateElectionDefinition(
+      renderer,
+      vxDefaultBallotTemplate,
+      // Each ballot style will have exactly one grid layout regardless of precinct, ballot type, or ballot mode
+      // So we just need to render a single ballot per ballot style to create the election definition
+      election.ballotStyles.map((ballotStyle) => ({
+        election,
+        ballotStyleId: ballotStyle.id,
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        precinctId: ballotStyle.precincts[0]!,
+        ballotType: BallotType.Precinct,
+        ballotMode: 'test',
+      })),
```

Comment:
> Should I move where the ballot images generated by generate-fixtures in
lib/hmpb to be in lib/fixtures? I think this might make sense in the long term
but I am trying to keep it out of scope for now.

I do think there could be some value in this. We could potentially just move the hmpb fixture generation into `libs/fixtures-generators`. But I think it can definitely be a separate project.

How are you planning to handle the many different elections in the hmpb fixtures (specifically in `general-election` where we test all the paper sizes)?